.github/workflows/lint.yml

```diff
@@ -0,0 +1,14 @@
+name: Markdown Lint
+on: [push, pull_request]
+jobs:
+  lint-changelog:
+    name: Lint changelog file
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Lint changelog file
+      uses: docker://avtodev/markdown-lint:v1
+      with:
+        args: '../CHANGELOG.md ../README.md ../CODE_OF_CONDUCT.md'
```

Comment:
why do we need to use a 3rd party action? Check the markdownlint-cli2 npm package

Reply:
@krishguptadev I didn't knew about that, I have looked up for `markdown lint` in GitHub marketplace.

Reply:
Changed the github action 👍